ibm/mas_devops/playbooks/ocp/configure-ocp.yml

```diff
@@ -2,6 +2,7 @@
 - hosts: localhost
   any_errors_fatal: true
   vars:
+    mas_channel: "{{ lookup('env', 'MAS_CHANNEL') | default('8.x', True)}}"
```

Comment:
Could we put a comment here explaining why the mas channel is needed .. I assume it's to determine how we configure the cluster, e.g. the comment should point the reader to where we can find details of how this is used so it can be easily found.

Reply:
Fixed.